src/tokens/selectors.ts

```diff
@@ -505,3 +506,20 @@ export const importedTokensSelector = createSelector(
     return tokenList.filter((token) => token?.isManuallyImported)
   }
 )
+
+const getJumpstartEnabledNetworkIds = () =>
+  Object.keys(
+    getDynamicConfigParams(DynamicConfigs[StatsigDynamicConfigs.WALLET_JUMPSTART_CONFIG])
+      .jumpstartContracts
+  ) as NetworkId[]
+
+export const jumpstartSendTokensSelector = createSelector(
+  [(state) => tokensWithTokenBalanceSelector(state, getJumpstartEnabledNetworkIds())],
+  (tokensWithBalance) => {
+    return tokensWithBalance.filter((token) => {
+      // the jumpstart contract currently requires a token address for the
+      // depositERC20 method
+      return !!token.address
```

Comment:
Not related to this PR, but I realized this whole flow won't work for depositing native ETH (it will work for WETH), unless we update the contract to support deposits in the native token.

Reply:
yes that is the whole reason for this selector :) otherwise i think we could reuse the send tokens selector